src/libYARP_OS/CMakeLists.txt

```diff
@@ -393,6 +404,10 @@ target_include_directories(YARP_OS PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOUR
 target_link_libraries(YARP_OS PRIVATE ${ACE_LIBRARIES}
                               PUBLIC YARP::YARP_conf)
 
+target_compile_features(YARP_OS PUBLIC cxx_nullptr
+                                       cxx_override
+                                       cxx_rvalue_references)
```

Comment:
`target_compile_features` is a CMake 3.1 command, so we should increase the `cmake_minimum_required` version to CMake 3.1 . Furthermore, as we still need to support Debian Jessie for the pc104 of several iCub robots, we should update the pc104 image (http://wiki.icub.org/wiki/The_Linux_on_the_pc104) to include CMake 3.6 from `jessie-backports`.